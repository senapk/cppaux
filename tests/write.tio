>>>>>>>>
========
5
4.5
[1, 2, 3]
[1.5, 2.5, 3.5]
(ovo, queijo)
(1, 2.3, tres)
sem quebrar linha ou mudando a quebra de linha.
[Joao:5, Maria:6]
A pessoa eh Joao:5.
<<<<<<<<
