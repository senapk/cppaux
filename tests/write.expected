5
4.5
[1, 2, 3]
[1.5, 2.5, 3.5]
(ovo, queijo)
(1, 2.3, tres)
5 eh cinco
5 nao eh seis
[Joao:5, Maria:6]
Joao:5
